packages/permissions/src/components/restricted-by-permissions/restricted-by-permissions.tsx

```diff
@@ -55,7 +55,7 @@ const RestrictedByPermissions = (props: Props) => {
       render={applicationContext => (
         <Authorized
           shouldMatchSomePermissions={props.shouldMatchSomePermissions}
-          actualPermissions={applicationContext.permissions}
+          actualPermissions={applicationContext.permissions || {}}
```

Comment:
Can you explain why this needs to change?

Reply:
I reverted this change 👍